modules/server/src/main/scala/scaladex/server/route/ProjectPages.scala

```diff
@@ -111,6 +111,37 @@ class ProjectPages(env: Env, database: WebDatabase, searchEngine: SearchEngine)(
           }
         }
       },
+
+      get {
+        path(projectM / "artifacts" ) { ref => 
+          getProjectOrRedirect(ref,user) { project => 
```

Comment:
To be able to filter on the binary versions you need to get the `ArtifactsPageParams` from the `<form>` in the front page. To do so you can call the `artifactsParams` method:

```suggestion
          artifactParams { params =>
            getProjectOrRedirect(ref,user) { project => 
```